torch/fx/operator_schemas.py

```diff
@@ -0,0 +1,85 @@
+import torch
+import inspect
+import numbers
+import typing
+import enum
+from typing import Any, Callable, Dict, List, Optional
+
+_manual_overrides : Dict[Callable, List[inspect.Signature]] = {}
+
+def _nonzero_schemas():
+    signatures = []
+
+    def nonzero(self):
+        pass
+    signatures.append(inspect.signature(nonzero))
+
+    def nonzero(self, *, as_tuple : bool):  # type: ignore
+        pass
+    signatures.append(inspect.signature(nonzero))
+
+    return signatures
+
+_manual_overrides[torch.nonzero] = _nonzero_schemas()
+
+class _FakeGlobalNamespace:
+    def __getattr__(self, name):
+        if name == 'torch':
+            return torch
+        raise RuntimeError('Expected a torch namespace lookup lookup')
+
+def _torchscript_type_to_python_type(ts_type : 'torch._C.JitType') -> Any:
+    g = {'Tensor' : torch.Tensor, 'Device' : torch.device, 'Layout' : torch.layout,
+         'number' : numbers.Number, 'Future' : torch.jit.Future,
+         'AnyEnumType' : enum.Enum, 'QScheme' : torch.qscheme,
+         '__torch__': _FakeGlobalNamespace(),
+         't': typing.TypeVar('t')}  # type: ignore
+    for k in dir(typing):
+        g[k] = getattr(typing, k)
+
+    return eval(ts_type.annotation_str, g)
```

Comment:
@eellison I used the `eval` trick you pointed out here. Can you take a look at the surrounding code and see if it looks reasonable?

Reply:
cc @gmagogsfm for review since this PR touches some of the TorchScript APIs and python bindings

Reply:
LGTM!